prep-javascript-functions/main.js

```diff
@@ -0,0 +1,62 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var addTwoNumbersResult = addTwoNumbers(5, 5);
+console.log('addTwoNumbers Exercise: ', addTwoNumbersResult);
+
+function convertHourstoMinutes(hours) {
+  return hours;
+}
+
+var convertHourstoMinutesResult = convertHourstoMinutes(5);
+console.log('convertHourstoMinutes Exercise: ', convertHourstoMinutesResult);
+
+function getGreeting(name) {
+  return name;
+}
+
+var getGreetingResult = getGreeting('World');
+console.log('Hello ' + getGreetingResult + '!');
+
+function addAndMultiplyBy5(num1, num2) {
+  return (num1 + num2) * 5;
+}
+
+var addAndMultiplyBy5Result = addAndMultiplyBy5(8, 5);
+console.log('addAndMultiplyBy5 Exercise: ', addAndMultiplyBy5Result);
+
+function multiplyAndDivideBy5(num1, num2) {
+  return (num1 * num2) / 5;
+}
+
+var multiplyAndDivideBy5Result = multiplyAndDivideBy5(45, 2);
+console.log('multiplyAndDivideBy5 Exercise: ', multiplyAndDivideBy5Result);
+
+function subtractTwoNumbers(num1, num2) {
+  return num1 - num2;
+}
+
+var subtractTwoNumbersResult = subtractTwoNumbers(33, 2);
+console.log('subtractTwoNumbers Exercise: ', subtractTwoNumbersResult);
+
+function getCircleCircumference(radius) {
+  return radius * 6.28318530718;
+}
+
+var getCircleCircumferenceResult = getCircleCircumference(7);
+console.log('getCircleCircumference Exercise: ', getCircleCircumferenceResult);
+
+function getFullName(firstName, lastName) {
+  return firstName + lastName;
+}
+
+var getFullNameResult = getFullName('Hannah ', 'Hoang');
+console.log('getFullName Exercise: ', getFullNameResult);
+
+function cube(number) {
+  return number * 4;
```

Comment:
this function needs to cube any number when it is called, not just the number 2.